### PR TITLE
Add missing license and manifests to the instrumentation packages.

### PR DESCRIPTION
**Pull request by mariusvniekerk**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Comment by ocelotl** [comment]
Thanks for the PR :v:

Please sign the CLA :slightly_smiling_face:

**Review by lonewolf3739 — APPROVED** [review]

File: instrumentation/opentelemetry-instrumentation-asgi/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,9 @@
+graft src
+graft tests
+global-exclude *.pyc
+global-exclude *.pyo
+global-exclude __pycache__/*
+include CHANGELOG.md
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-asgi/MANIFEST.in:6]: Do we need this line? I don't see it in this package. Same for all the other `MANIFEST.in` files?

File: instrumentation/opentelemetry-instrumentation-asgi/MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,9 @@
+graft src
+graft tests
+global-exclude *.pyc
+global-exclude *.pyo
```
Comment (NathanielRN) [inline instrumentation/opentelemetry-instrumentation-asgi/MANIFEST.in:4]: Not sure what kind of files have these extensions 😅 But looks like we do it everywhere so can't be too much harm to have this?

**Review by NathanielRN — APPROVED** [review]
Thanks for doing this! Added a comment about the `CHANGELOG.md` file but otherwise looks great 🙂

**Review by codeboten — COMMENTED** [review]
Thanks for the PR @mariusvniekerk, please sign the CLA so we can get this merged.

**Comment by owais** [comment]
@mariusvniekerk Thanks for the contribution. This is ready to go but we need you to sign the CLA before we can merge :) 

Check this comment for details on how to sign: https://github.com/open-telemetry/opentelemetry-python-contrib/pull/616#issuecomment-896843340

**Comment by ocelotl** [comment]
@owais, @lzchen this is a good contribution that has been sitting here for a while now. Would you be ok with merging it even without a CLA?

**Comment by ocelotl** [comment]
> @owais, @lzchen this is a good contribution that has been sitting here for a while now. Would you be ok with merging it even without a CLA?

We have decided not to merge without a CLA.

**Comment by lzchen** [comment]
@mariusvniekerk

Last ping before we decide to close/take over this PR. Please fix the build so we can merge this in.

**Comment by lzchen** [comment]
@mariusvniekerk 
Closing this PR for now. Feel free to comment/open it back up if you resume work on it.

**Pull request closed without merging**